### PR TITLE
ui. fix spam filter checkbox and rspamd rules removal

### PR DESCRIPTION
**Pull request by federicoballarini**
Fixed Spam Filter checkbox that wasn't shown properly.
Refer to https://github.com/NethServer/dev/issues/5952

### CODE REVIEW

**Comment by federicoballarini** [comment]
Fixed deleting rspamd rules from Cockpit. Refer to https://github.com/NethServer/dev/issues/5951

_Note: sorry for branch mistake. From next PR i will separate them._

File: ui/src/views/Connectors.vue [hunk 1]
```diff
@@ -680,8 +674,7 @@ export default {
     },
     openEditConnector(conn, email) {
       this.newConnector = Object.assign({}, conn);
-      this.newConnector.props.FilterCheck =
-        this.newConnector.props.FilterCheck == "enabled";
+      this.newConnector.props.FilterCheck = this.newConnector.props.FilterCheck == "enabled";
```
Comment (gsanchietti) [inline ui/src/views/Connectors.vue]: Please avoid commits only for the code indentation because they really create a bad diff which is hard to review.
If a code indentation fix is really needed, move it to a different commit.

File: ui/src/views/Connectors.vue [hunk 2]
```diff
@@ -344,13 +344,7 @@
               >
                 <label class="col-sm-3 control-label">{{$t('connectors.filtercheck')}}</label>
                 <div class="col-sm-9">
-                  <input
```
Comment (gsanchietti) [inline ui/src/views/Connectors.vue]: If possible, try to not change the logic.

In this case, make sure the code which reads the value just return `enabled` and `disabled` and not the boolean value.

File: ui/src/views/Filter.vue [hunk 3]
```diff
@@ -938,10 +938,19 @@ export default {
...
+       for(var i = 0; i < this.filter.WBList.length; i++){
+            var x = this.filter.WBList[i];
+            if(x['type'] == obj.type && x['value'] == obj.value){
+                rulesindex = i;
+                i = this.filter.WBList.length;
+            }
+        }
+        
+        this.filter.WBList.splice(rulesindex, 1);
+        $("#deleteRuleModal").modal("hide");
```
Comment (gsanchietti) [inline ui/src/views/Filter.vue]: This modification is not relevant to the describe issue.
Please remove it from the PR.

File: ui/src/views/Filter.vue [hunk 4]
```diff
@@ -938,10 +938,19 @@ export default {
...
 
-      this.saveRules();
+        this.saveRules();
```
Comment (gsanchietti) [inline ui/src/views/Filter.vue]: Avoid useless space changes.

**Comment by federicoballarini** [comment]
Thanks Giacomo for your time. I will close this PR and open a new one because I make a mistake with branches.
